docs/Configuration.rst

```diff
@@ -469,24 +511,42 @@ template value here)::
 Optional configuration settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Software and modules install path suffixes
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _prefix:
+
+Overall prefix path (``--prefix``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*default:* ``$HOME/.local/easybuild``
 
-This section describes configuration options
-``--subdir-software``, ``--subdir-modules``, ``--suffix-modules-path``,
-which are supported since v1.14.0.
+The overall prefix path used by EasyBuild can be specified using the ``--prefix`` configuration option.
 
-*defaults*: ``software`` as software install path suffix, ``modules`` as
-modules install path suffix
+This affects the default value of several configuration options:
 
-The software and modules install path suffixes can be adjusted using the
-``subdir-software`` and/or ``subdir-modules`` / ``suffix-modules-path`` configuration settings,
-for example:
+* source path (see :ref:`sourcepath`)
+* build path (see :ref:`buildpath`)
+* software and modules install path (see :ref:`installpath`)
+* easyconfigs repository path (see :ref:`easyconfigs_repo`)
+
+.. _installpath_subdirs:
+
+Software and modules install path subdirectories (``--subdir-software``, ``--subdir-modules``, ``--suffix-modules-path``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*defaults*:
+
+* *software install path subdirectory* (``--subdir-software``): ``software``
+* *modules install path subdirectory* (``--subdir-modules``): ``modules``
+* *modules install path suffix* (``--suffix-modules-path``): ``all``
+
+The subdirectories for the software and modules install paths (relative to ``--installpath``, see :ref:`installpath`)
+can be specified using the corresponding dedicated configuration options (available since EasyBuild v1.14.0).
```

Comment:
I'm not dropping it anywhere, this corresponds to the line being dropped that mentions v1.14.0. I just added an additional section, so the diff is a bit weird.